test/Microsoft.ML.CodeGenerator.Tests/ApprovalTests/ConsoleCodeGeneratorTests.cs

```diff
@@ -308,9 +308,10 @@ private CodeGenerator PrepareForRecommendationTask()
                 var textLoaderArgs = new TextLoader.Options()
                 {
                     Columns = new[] {
-                        new TextLoader.Column("Label", DataKind.String, 0),
-                        new TextLoader.Column("userId", DataKind.String, 1),
-                        new TextLoader.Column("movieId", DataKind.String, 2),
+                        new TextLoader.Column("userId", DataKind.Single, 0),
+                        new TextLoader.Column("movieId", DataKind.Single, 1),
+                        new TextLoader.Column("rating", DataKind.Single, 2),
+                        new TextLoader.Column("timestamp", DataKind.Single, 3),
```

Comment:
Why load this column here, only to ignore it later?

Reply:
Only because the sample in machinelearning-samples repo already has that column and I want to show it can be used and ignored at the same time.

I would have not included this if the existing dataset did not come with that column. It's added just for convenience of not wanting to add a whole new dataset to just make this work. (easier to later compare with FFM and to currently compare with the sample in machinelearning-samples repo), since they all use the same dataset.